src/PKSim.Core/Snapshots/Mappers/DataColumnMapper.cs

```diff
@@ -67,10 +67,10 @@ public override async Task<ModelDataColumn> MapToModel(SnapshotDataColumn snapsh
          var dataInfo = await _dataInfoMapper.MapToModel(snapshot.DataInfo, snapshotContext);
          var dimension = dimensionFrom(snapshot);
          var dataColumn = dataInfo.Origin == ColumnOrigins.BaseGrid ? new BaseGrid(snapshot.Name, dimension) : new ModelDataColumn(snapshot.Name, dimension, dataRepository.BaseGrid);
+         //this needs to be set after DATA Info to be sure that we are using the value from the snapshot
+         dataColumn.DataInfo = dataInfo;
          dataColumn.DisplayUnit = displayUnitFor(dimension, snapshot.Unit);
          dataColumn.Values = valuesInBaseUnits(dataColumn, snapshot.Values);
-
-         dataColumn.DataInfo = dataInfo;
```

Comment:
very interesting side effects. DisplayUnit is saved in the DataInfo and not in the column. Therefor, by setting the display unit AND THEN the dataInfo, it was reset to the default (which in our case is not serialized in the snapshot)
So it is done the way it should de in snapshot. Unit belongs with the Data and not somewhere else. (WTH !!)